packages/mongodb-memory-server-core/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "rootDir": "./src",
-    "outDir": "./lib",
+    "outDir": "./build",
```

Comment:
Why do you change `outDir` in `tsconfig.json` to `./build`?

Firstly I had thought that this is an error, and the core package will be with empty `/lib` folder, but after that, I saw that you override it back to `./lib` in `tsconfig.build.json`.

Reply:
because this is the config, where everything `.ts` is included (like source AND Tests), but because we dont want the tests in the published build (and because we already had and tsconfig.build) i changed it in this one to `build` (so that it is not going into the same folder)